sql/driver/wire.go

```diff
@@ -53,9 +54,8 @@ func makeDatum(val driver.Value) (Datum, error) {
 		datum.StringVal = &t
 	case time.Time:
 		// Send absolute time devoid of time-zone.
-		datum.TimeVal = &Datum_Timestamp{
-			Sec:  t.Unix(),
-			Nsec: uint32(t.Nanosecond()),
+		datum.TimestampVal = &proto.Timestamp{
+			WallTime: t.UnixNano(),
```

Comment:
This isn't the same at all. `time.Time` is a 96-bit value composed of 64-bit seconds and 32-bit nanoseconds field. This compresses it down to a 64-bit value losing tons of range. The 64-bit nanosecond field used by `proto.Timestamp` is fine for our internal purposes (a range of +/- 292 years from 1970), but not for representing a `time.Time`.